### PR TITLE
Added spawned entity class vars

### PR DESCRIPTION
**Pull request by fruitwasp**
Cleaner than Entity:GetClass() == "x"

### CODE REVIEW

**Comment by FPtje** [comment]
Good idea, but shouldn't those changes resonate in the rest of DarkRP's code?

**Comment by fruitwasp** [comment]
Will do.

**Comment by fruitwasp** [comment]
Commit squash failed. Sorry about that.

File: entities/entities/spawned_money/init.lua [hunk 1]
```diff
@@ -30,7 +30,7 @@ end
 
 function ENT:Touch(ent)
 	-- the .USED var is also used in other mods for the same purpose
-	if ent:GetClass() ~= "spawned_money" or self.USED or ent.USED or self.hasMerged or ent.hasMerged then return end
+	if not ent.IsSpawnedMoney or self.USED or ent.USED or self.hasMerged or ent.hasMerged then return end
```
Comment (FPtje) [inline entities/entities/spawned_money/init.lua]: We might want to enforce that only two spawned_money can merge. That makes most sense and doesn't have the risk of errors with other money kinds that use different variables.

Comment (fruitwasp) [inline entities/entities/spawned_money/init.lua]: Now, we're talking about that function anyway. Change the function to [ENT:StartTouch](http://wiki.garrysmod.com/page/ENTITY/StartTouch), so it's not called continuously?

Comment (FPtje) [inline entities/entities/spawned_money/init.lua]: sure

**Pull request merged**